package.json

```diff
@@ -6,7 +6,9 @@
   "private": true,
   "scripts": {
     "a11y": "yarn workspace client lighthouse",
-    "dev": "docker-compose up",
+    "dev": "npm-run-all --parallel dev:*",
+    "dev:client": "yarn workspace client run dev",
+    "dev:server": "yarn workspace server run dev",
     "jest": "yarn workspaces run jest",
     "jestAll": "docker-compose -f docker-compose.yml -f docker-compose.jest.yml up",
```

Comment:
What about this docker command? There is documentation for this in local development file as well.